x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_style_property.js

```diff
@@ -151,7 +143,7 @@ export class DynamicStyleProperty extends AbstractStyleProperty {
 
   supportsFieldMeta() {
     if (this.isOrdinal()) {
-      return this.isComplete() && this.isOrdinalScaled() && this._field.supportsFieldMeta();
+      return this.isComplete() && this._field.supportsFieldMeta();
```

Comment:
Removing `isOrdinalScaled` check has introduced a regression. `DynamicColorProperty.isOrdinalScaled` checked that `!this.isCustomOrdinalColorRamp()` so meta was not requested for custom color ramps because its not needed. However, now meta is requested for custom color ramps.

The gear Icons should not be displayed in the image below.

<img width="394" alt="Screen Shot 2020-03-19 at 6 41 15 AM" src="https://user-images.githubusercontent.com/373691/77068771-c2b2a380-69ac-11ea-8c1b-aa6e04a04cbe.png">

Inspector should only show a single requests.

<img width="490" alt="Screen Shot 2020-03-19 at 6 39 11 AM" src="https://user-images.githubusercontent.com/373691/77068743-b29ac400-69ac-11ea-9df8-dc9ede84ae04.png">

I think the solution will be that DynamicColorProperty will have to override `supportsFieldMeta `. When fixing this, mind adding a unit test to DynamicColorProperty to verify that `supportsFieldMeta` returns false for custom color ramps?

If the solution is to override `supportsFieldMeta` in DynamicColorProperty, then supportsFieldMeta in the super class can be simplified to just `return this.isComplete() && this._field.supportsFieldMeta();` since both ordinal and categorical branches are the same now.